{
 "name": "ConnectionError",
 "fields": [
  {
   "containsDiscriminatedTypeValue": false,
   "jsonName": "connectionState",
   "name": "ConnectionState",
   "objectDefinition": {
    "type": "Reference",
    "referenceName": "ConnectionState"
   },
   "optional": true,
   "required": false,
   "description": "The kind of connection error that occurred."
  },
  {
   "containsDiscriminatedTypeValue": false,
   "jsonName": "exception",
   "name": "Exception",
   "objectDefinition": {
    "type": "String",
    "referenceName": null
   },
   "optional": true,
   "required": false,
   "description": "Detailed error message about the failed connection."
  },
  {
   "containsDiscriminatedTypeValue": false,
   "jsonName": "iPFrom",
   "name": "IPFrom",
   "objectDefinition": {
    "type": "String",
    "referenceName": null
   },
   "optional": true,
   "required": false,
   "description": "The IP of host that originated the failed connection."
  },
  {
   "containsDiscriminatedTypeValue": false,
   "jsonName": "iPTo",
   "name": "IPTo",
   "objectDefinition": {
    "type": "String",
    "referenceName": null
   },
   "optional": true,
   "required": false,
   "description": "The IP that the connection attempted to reach."
  },
  {
   "containsDiscriminatedTypeValue": false,
   "jsonName": "port",
   "name": "Port",
   "objectDefinition": {
    "type": "Integer",
    "referenceName": null
   },
   "optional": true,
   "required": false,
   "description": "The TCP port the connection was attempted on."
  }
 ]
}